### ISSUE TITLE
AttributeError No attribute 'decodebytes' on knitpy.render()

### ISSUE THREAD

**Issue by nehalecky** [JanSchulz/knitpy#41]
I can confirm that that #38 fixed the other issues, thank you much @JanSchulz.

Again, hope this issue reporting is beneficial. :)

In the call to `.render` :
```python
In [14]: knitpy.render('knitpy_overview.pymd')
[TerminalIPythonApp] ERROR | Could not save a image
Traceback (most recent call last):
  File "/home/vagrant/vm/shared/knitpy/knitpy/documents.py", line 292, in add_image
    mimedata = base64.decodebytes(mimedata.encode())
AttributeError: 'module' object has no attribute 'decodebytes'
Out[14]: [u'/home/vagrant/vm/shared/knitpy/examples/knitpy_overview.html']
```
The call does generate the resultant documents, so that's great. How else can I help?

**Comment by JanSchulz** [comment]
I do hope I got that... I really need to get that unittesting going...

**Comment by JanSchulz** [comment]
Please checkout master again, this should be fixed...

**Comment by nehalecky** [comment]
Hi @JanSchulz, thank you! Yes, fixed via 265cf1773ba0e51a636efccbe0ad0d3c0f48c3e8
